Source/WebCore/accessibility/AccessibilityObject.cpp

```diff
@@ -3739,9 +3739,24 @@ Vector<Element*> AccessibilityObject::elementsFromAttribute(const QualifiedName&
     if (!node || !node->isElementNode())
         return { };
 
+    auto& element = downcast<Element>(*node);
+    if (Element::isElementReflectionAttribute(attribute)) {
+        if (auto reflectedElement = element.getElementAttribute(attribute)) {
+            Vector<Element*> elements;
+            elements.append(reflectedElement);
+            return elements;
+        }
+    } else if (Element::isElementsArrayReflectionAttribute(attribute)) {
+        if (auto reflectedElements = element.getElementsArrayAttribute(attribute)) {
+            Vector<Element*> elements;
+            for (auto reflectedElement : reflectedElements.value())
```

Comment:
Use WTF::map?

Reply:
Good idea. Used it also a few lines below too.